generators/cleanup.js

```diff
@@ -255,6 +255,8 @@ function cleanupOldFiles(generator) {
         } else if (generator.jhipsterConfig.clientFramework === REACT) {
             generator.removeFile(`${REACT_DIR}modules/administration/audits/audits.tsx`);
             generator.removeFile(`${CLIENT_TEST_SRC_DIR}spec/enzyme-setup.ts`);
+            generator.removeFolder(`${CLIENT_TEST_SRC_DIR}spec`);
+            generator.removeFile(`${CLIENT_TEST_SRC_DIR}jest.conf.js`);
```

Comment:
2 comments here:
* cleanup should go under `7.0.0-beta.1` section to clean up files for users using version `7.0.0-beta.0`
* as per https://github.com/jhipster/generator-jhipster/pull/13171#issuecomment-736543928 it's better to remove file by file, not entire folder

Reply:
ok

Reply:
Entity tests also need to be cleaned up, look at https://github.com/jhipster/generator-jhipster/blob/main/generators/entity-client/index.js#L70

Reply:
done